pkg/transportrequest/gitutils.go

```diff
@@ -2,12 +2,73 @@ package transportrequest
 
 import (
 	"fmt"
+	gitUtils "github.com/SAP/jenkins-library/pkg/git"
```

Comment:
What about keeping this out of this PR here? That PR could focus only of providing the functionality for opening a git repo in our git utils. When more things are kept out we can focus the review which will come with less efforts.

Reply:
gitutils is the reason we add plainOpen() for. it helps to understand why and how to implement it.

Reply:
OK, we have maybe different approach when working with PRs. We can also continue with the "big approach". I personally think that having it splitted in several PRs makes it easier, but that is only my personal experience and maybe I'm wrong ...

Reply:
I'd say the PR is still small, but in case it evolves differently, I split.